content/blog/2020-03-19-howto/index.fr.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Comment proposer un article
+author: Samuel Perreault et David Beauchemin
+date: '2020-04-21'
+slug: howto-fr
+type: post
+categories: ["Contribuer"]
+tags: []
+description: "Un gabarit pour contribuer au blog .Layer"
+featured: "howto-fr-cover.png"
+featuredpath: "img/headers/"
+---
+
+Contribuer au blog n'aura jamais été aussi facile. Tout d'abord, il faut dire que toute soumission, quel que soit son format (Markdown, Microsoft Word, Notepad, *name it*!) sera considérée, et ultimement transcrite en Markdown. On pense déjà à une façon de faire pour la révision des documents non-Markdown (possiblement Google Docs). Ceci étant écrit, pour ceux et celles qui voudraient écrire et soumettre un article de la façon _conventionnelle_, voici une procédure simple pour y arriver.
+
+1. **Création de l'article**
+  1. Enregister le fichier `.md` utilisé pour créer cet article (disponible [ici](https://github.com/dot-layer/blog/blob/master/content/blog/2020-03-19-howto/index.fr.md)) sous le nom `index.md` (pour les articles en anglais) ou `index.fr.md` (pour les articles en français).
```

Comment:
```suggestion
  1. Enregiste le fichier en format `.md` utilisé pour créer cet article (disponible [ici](https://github.com/dot-layer/blog/blob/master/content/blog/2020-03-19-howto/index.fr.md)) sous le nom `index.md` (pour les articles en anglais) ou `index.fr.md` (pour les articles en français).
```

Reply:
Les autres "bullet" commencent avec un verbe à l'infinitif (comme une recette). Je garderais le "er" là (et plus bas). C'est le dernier truc à settle après on est good to go!